[SendGrid LIsts] - updating some wording (#7370)
segmentio/segment-docs: src/connections/destinations/catalog/actions-sendgrid-audiences/index.md

 - Phone Number ID (must be in [E.164](https://www.twilio.com/docs/glossary/what-e164){:target="_blank”} format)
 - External ID 
 

> Sending more than one identifier should only be done with caution: If you provide more than one identifier, SendGrid will require all of those identifiers for future updates to that Contact.

Reviewer: ```suggestion
> warning ""
> If you provide more than one type of identifier for each user in your initial sync, SendGrid requires you to send all of those identifier types for any future updates to that Contact.
```

> To sync Engage users to a SendGrid list using an identifier type other than email, additional steps may be required:

Reviewer: ```suggestion
To sync Engage users to a SendGrid list using an identifier type other than email, complete the following additional steps:
```

1. Configure [ID Sync](/docs/engage/trait-activation/id-sync/) to include a value for the identifier when syncing users from an Engage Audience to the SendGrid List. 

> 2. Map the identifier using the correct [Sync Audience ](#sync-audience-action) Action's mapping field.  

Reviewer: ```suggestion
2. Map the identifier using the [Sync Audience Action](#sync-audience-action)'s mapping field.  
```
